gripper_joint_state_publisher:
  ros__parameters:
    joint_names:
      - "panda_finger_joint1"
      - "panda_finger_joint2"
